mock-integrations/mock-openapi.yaml

```
@@ -162,7 +162,7 @@ paths:
                         id: 25
                         middlenames: ""
                         salutation: Mr
-                        surname: Matthews
+                        surname: M’atthews
```

```suggestion
                        surname: Matthews
```
Probably only need the one example in the file